### PR TITLE
Add LCOV plugin to default channel.

### PR DESCRIPTION
**Pull request by thraex-aquator**
- [x] I'm the package's author and/or maintainer.
- [x] I have have read [the docs][1].
- [x] I have tagged a release with a [semver][2] version number.
- [x] My package repo has a description and a README describing what it's for and how to use it.
- [x] My package doesn't add context menu entries. *
- [x] My package doesn't add key bindings. **

My package is showing covered/missed lines based on an lcov.info file.

There are no packages like it in Package Control.

[1]: https://packagecontrol.io/docs/submitting_a_package
[2]: https://semver.org
[3]: https://www.git-scm.com/docs/gitattributes#_export_ignore

### CODE REVIEW

**Comment by kaste** [comment]
LGTM from here.  I think it requires Sublime Text `>=4000` though as you target (rightfully) python 3.8.  Note that you don't need to instantiate `Listener()`; Sublime does it for you automatically.

**Comment by thraex-aquator** [comment]
Thanks for the heads-up! Bumped the version to 4000, also removed the extra initialization and reduced the scope of the observer while I was already there. Moved the 1.0.0 tag to the current head as it was never really released before.

**Review by kaste — APPROVED** [review]

File: repository/l.json [hunk 1]
```diff
@@ -673,6 +673,18 @@
 				}
 			]
 		},
+		{
+			"name": "LCOV",
+			"details": "https://gitlab.com/aquator/sublime-lcov",
+			"readme": "https://gitlab.com/aquator/sublime-lcov/-/raw/master/README.md",
```
Comment (braver) [inline repository/l.json:679]:
```suggestion
```

Since that's just the default readme location, you can omit it.

**Comment by braver** [comment]
Ugh, our tools don't want to run on GitLab repos. It looks good enough to just run with it and see what happens though 👍🏻

File: repository/l.json (hunk 1 above)
Comment (thraex-aquator) [reply repository/l.json]: Thanks, did not know that, I copied from another gitlab based plugin.
Removed the readme line.

**Comment by thraex-aquator** [comment]
Do I need to do anything else regarding being on gitlab, or you will handle it? (The suggested change is there)

**Comment by braver** [comment]
Ok, let’s see if it works. Maybe we need to add some metadata here, but we just need to check in a couple of hours.

**Comment by kaste** [comment]
✋ packagecontrol.io doesn't pick it up

**Comment by braver** [comment]
@kaste thanks reporting that. It seems to be a more widespread issue than just this package. I have no idea what might cause it. Of 5516 packages, 6 are on GitLab, half of which are missing. 🤷🏻 

Will look into it: https://github.com/wbond/package_control_channel/issues/9020

**Pull request merged**